src/server/src/main/java/io/cassandrareaper/jmx/ClusterFacade.java

```diff
@@ -53,12 +55,23 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.Lists;
+import jersey.repackaged.com.google.common.base.Preconditions;
 import org.apache.commons.lang3.tuple.Pair;
+import org.joda.time.DateTime;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @author adejanovski
+ *
+ */
+/**
+ * @author adejanovski
+ *
+ */
```

Comment:
remove author tags (especially duplicates :-)

Reply:
has this been fixed? (i'm still seeing duplicate author tags in github)

Reply:
Right, it's still there. Removing it :)